docs/content/latest/releases/earlier-releases/v2.1.5.md

```diff
@@ -0,0 +1,100 @@
+---
+title: What's new in 2.1.5
+headerTitle: What's new in 2.1.5
+linkTitle: What's new in 2.1.5
+description: Enhancements, changes, and resolved issues in the latest YugabyteDB release.
```

Comment:
this description has to change as well to match that this is now a specific earlier release (similar to how other earlier releases are handled)

Reply:
Fixed.